roles/openshift_master/tasks/main.yml

```diff
@@ -76,14 +77,14 @@
   args:
     creates: "{{ openshift_master_policy }}"
   notify:
-  - restart openshift-master
+  - restart {{ product_type }}-master
```

Comment:
The handler name is `restart master` and the service name is extracted there. Replace all instances of `restart {{ product_type }}-master` with `restart master`  should be 3 of them.